### ISSUE TITLE
Cyrillic review

### ISSUE THREAD

**Issue by vv-monsalve** [TypeTogether/Playpen-Sans#20]
Hi, @josescaglione, I'm bringing here the feedback provided by @alexeiva on Cyrillic.

His main comment was:

> Bulgarian:
missing glyphs:
ѣ ѫ
They are necessary for writing historical pre-1923 text. This may not be very useful today.

[replaypensansi18nreview.zip](https://github.com/user-attachments/files/17105288/replaypensansi18nreview.zip)

These PDFs correspond to the round 1 proofs produced at commit `e6f01d6 fix an anchor`.

Some reported issues might have already been solved since you continued working on the fonts.

**Comment by josescaglione** [comment]
the two glyphs mentioned above were removed from this set in February. They are for old Slavonic church texts.

**Comment by alexeiva** [comment]
> the two glyphs mentioned above were removed from this set in February. They are for old Slavonic church texts.

Yes, correct. Also glyphs `Ѐ Ѣ Ꙝ Ѥ Ѫ Ѭ` were used in pre-1945 Bulgarian texts, that's why they are included in the Bulgarign AUX set. It is fine not to include them in a handwritten font.

**Comment by vv-monsalve** [comment]
@josescaglione Following the latest definition of the [GF_Cyrillic_Core](https://github.com/googlefonts/glyphsets/blob/main/GLYPHSETS.md#gf-cyrillic-core) and [GF_Cyrillic_Plus](https://github.com/googlefonts/glyphsets/blob/main/GLYPHSETS.md#gf-cyrillic-plus) Glyphsets, the following are the reported missing glyphs.

<details><summary><b>GF Cyrillic Core</b></summary>
- Imacron-cy
- Umacron-cy
- imacron-cy
- umacron-cy

</details>

<details><summary><b>GF Cyrillic Plus</b></summary>

- Gemiddlehook-cy
- Abreve-cy
- Iebreve-cy
- Zhedieresis-cy
- Zedieresis-cy
- Idieresis-cy
- Odieresis-cy
- Udieresis-cy
- Uhungarumlaut-cy
- Chedieresis-cy
- gemiddlehook-cy
- abreve-cy
- iebreve-cy
- zhedieresis-cy
- zedieresis-cy
- idieresis-cy
- odieresis-cy
- udieresis-cy
- uhungarumlaut-cy
- chedieresis-cy

</details>

Most of them are composites that would only need good anchors.

<img width="1207" alt="Screenshot 2024-10-17 at 16 15 46" src="https://github.com/user-attachments/assets/13a23446-d4d1-416b-9e8f-d867ac7d98ff">

**Comment by alexeiva** [comment]
Hi @josescaglione @vv-monsalve 

I have reviewed round 2. All the base Cyrillic Plus glyphs are now intact. 
There are some kerning issues and collisions:


[review-2.pdf](https://github.com/user-attachments/files/17585341/review-2.pdf)

**Comment by josescaglione** [comment]
Hi @alexeiva 

Thank you so much for the detailed feedback, it was extremely useful. I wanted to point out that in this typeface we have opted to allow some clashes in favor of the texture continuity and casual looks. However, we have tweaked most of the pairs you pointed out and will be visible at the next commit.

As regards the i-cy yi-cy clash, given the technical issues related to the shuffler and compounds being generated on the fly, we will try to sort it out by means of a ligature.
Could you please confirm that these letters can appear in any order:
 i-cy yi-cy
yi-cy  i-cy 


Thanks,
J

**Comment by alexeiva** [comment]
creating a `yi-cy_yi-cy` ligature would be an elegant solution

її (yi-cy_yi-cy`) — is a Ukrainian pronoun

> Could you please confirm that these letters can appear in any order:

**i-cy yi-cy**
yes. Common word ending in Ukrainian (-ії), and female name Ії (I-cy yi-cy)

**yi-cy i-cy**
no. This isn't used in Ukrainian

**Comment by josescaglione** [comment]
Thanks @alexeiva !!

**Comment by alexeiva** [comment]
Considering this new addition on local variants to the glyph set https://github.com/googlefonts/glyphsets/pull/239

It would be good to also have these Bashkir local variants:

```
Gestroke-cy.loclBSH
Zedescender-cy.loclBSH
Esdescender-cy.loclBSH
gestroke-cy.loclBSH
esdescender-cy.loclBSH
```

Here is an illustration of how there are drawn. 
[review-CYR-locl.pdf](https://github.com/user-attachments/files/17597777/review-CYR-locl.pdf)

It is fine to use the roman(marked blue) style instead of handwritten. The 'roman' style will match better with қ,ң,ц,щ, etc


<img width="916" alt="Screenshot 2024-11-01 at 13 39 17" src="https://github.com/user-attachments/assets/474e09ea-edb4-429f-b454-240ee2be5383">

**Comment by josescaglione** [comment]
> creating a `yi-cy_yi-cy` ligature would be an elegant solution
> 
> її (yi-cy_yi-cy`) — is a Ukrainian pronoun
> 
> > Could you please confirm that these letters can appear in any order:
> 
> **i-cy yi-cy** yes. Common word ending in Ukrainian (-ії), and female name Ії (I-cy yi-cy)

@vv-monsalve this is sorted out in the latest version. Also fixed many of the kerning pairs mentioned by @alexeiva 
<img width="543" alt="Screen Shot 2024-11-04 at 13 19 50" src="https://github.com/user-attachments/assets/2082f779-8ccc-4ef4-a1ec-678fcbfa3f29">

**Comment by alexeiva** [comment]
@josescaglione 

`ї + ї` with displaced three dots is a recent innovation (similar to Cap Eszett), there are still debates on its use. I want to point out that since it is used only in a separate pronoun`її` — there is no issue with ambiguity. And I know many native designers who back this idea too. I would say it is safe to use it here.

**Comment by josescaglione** [comment]
@alexeiva 
We used a similar feature in Latin for the double idieresis in Dinka language. So far we got no complains which is good news :)

**Comment by vv-monsalve** [comment]
@josescaglione 

Here a comparison image of the kerning cases reported by Alexei. Please review them and ensure everything is working as intended. There is a regression in the pairs `Tï` and `Tî` 

![Image](https://github.com/user-attachments/assets/c2b97036-0e67-46d6-8c6c-10d148d61f0a)


Tested string
````
Tï Tõ włw yły 74 
Еи мҷ мч УЈ Пс 
p̱ara p̱o ïï гъ қәқл in ҳуқуқларни кл in шаклланиши Ал in Александар үр in хүргэж АД in РАДОВАО ҲД 
in ФҲДЁ қа in қарарin
Ұл in Ұлттар ДУ къ in където тэ in тэжээхэд АД in АДАДАДАД қо in қолиб қу in қурувчиси қр in очиқроқ тх in ибодатхонасида
қа in чиққан тт in тттттт
тт ал ал Александар ар съкровения үзэж гтгтгт кредитования кркркркркркр человек радује АДАДАДАД ïï ҳуқуқларни чиққан ФҲДЁ
Ti Tí Tî Tï Tì Tī Vă Tä Tü Tõ Tŭ Tř
````

**Comment by josescaglione** [comment]
This is correct, the new version removes kerning because of the exceptions problem we already discussed, and incorporates the idieresis ligature.
